src/components/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+// eslint-disable-next-line
+const links = [
+  {
+    id: 1,
+    path: '/',
+    text: 'Home',
+  },
+  {
+    id: 2,
+    path: '/about',
+    text: 'About',
+  },
+];
+
+const Navbar = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  const handleToggle = () => {
+    setNavbarOpen((prev) => !prev);
+  };
+
+  const closeMenu = () => {
+    setNavbarOpen(false);
+  };
+
+  return (
+    <nav className="navBar">
+      <button onClick={handleToggle} type="button">
+        {navbarOpen ? (
+          <MdClose style={{ color: '#fff', width: '40px', height: '40px' }} />
+        ) : (
+          <FiMenu style={{ color: '#7b7b7b', width: '40px', height: '40px' }} />
+        )}
+      </button>
+      <ul className={`menuNav ${navbarOpen ? ' showMenu' : ''}`}>
+        {links.map((link) => (
+          <li key={link.id}>
+            <NavLink
+              to={link.path}
+              activeClassName="active-link"
```

Comment:
Please consider fixing this error which displays on the console. 

![image](https://user-images.githubusercontent.com/116313214/212915103-2d248832-8f60-4abe-9b02-fa88572adedd.png)

This can be fixed by changing this line of code 👍